skuba-update/test/os/suse/suse.sh

```diff
@@ -116,6 +116,7 @@ check_kubectl_calls() {
         for i in "$@"; do
             echo "$i" >> /commands.txt
         done
+        command -v diff || zypper install -y diffutils
```

Comment:
there is a install_package() wrapper that calls zypper with the right arguments (namely -n). 

so suggest to use

```bash
 command -v diff || install_package diffutils
```

Reply:
meh that will reinstall other package by default...